src/code/z_rcp.c

```diff
@@ -1246,23 +1246,33 @@ Gfx* Gfx_SetupDL_69NoCD(Gfx* gfx) {
     return gfx;
 }
 
+#if OOT_DEBUG
+#define HREG_21 HREG(21)
+#define HREG_22 HREG(22)
+#else
+#define HREG_21 0
+#define HREG_22 0
+#endif
+
 Gfx* func_800947AC(Gfx* gfx) {
     gSPDisplayList(gfx++, sSetupDL[SETUPDL_65]);
     gDPSetColorDither(gfx++, G_CD_DISABLE);
 
     // clang-format off
-    switch (HREG(21)) {
+    switch (HREG_21) {
```

Comment:
I think I prefer the ternary actually, IMO it's a bit easier to read since my eyes don't have to jump somewhere else to see what's going on. But I don't feel strongly about it so feel free to ignore